gto/registry.py

```diff
@@ -314,7 +314,7 @@ def assign(  # pylint: disable=too-many-locals
             and any(vstage.stage == stage for vstage in found_version.get_vstages())
         ):
             raise WrongArgs(
-                f"Version '{found_version.version}' is already in stage '{stage}. Use the '--force' flag to reassign the '{stage}' stage to version '{found_version.version}"
+                f"Version '{found_version.version}' is already in stage '{stage}. Use the '--force' to create new Git tag nevertheless."
```

Comment:
I'm all for Star Wars references - but should it be:

`Use the '--force' flag to create a new Git tag nevertheless.`

Reply:
ahh, merged it too early